scripts/groups/restore_groups.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/ONSdigital/dp-identity-api/models"
+	"github.com/ONSdigital/dp-identity-api/scripts/utils"
+	"github.com/ONSdigital/log.go/v2/log"
+	cognito "github.com/aws/aws-sdk-go/service/cognitoidentityprovider"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+)
+
+type config struct {
+	groupsFilename       string `envconfig:"groups_filename"`
+	groupUsersFilename   string `envconfig:"groupusers_filename"`
+	s3Bucket             string `envconfig:"s3_bucket"`
+	s3BaseDir            string `envconfig:"s3_base_dir"`
+	s3Region             string `envconfig:"s3_region"`
+	awsCognitoUserPoolID string `envconfig:"user_pool_id"`
+}
+
+func (c config) getS3GroupsFilePath() string {
+	return fmt.Sprintf("%s%s", c.s3BaseDir, c.groupsFilename)
+}
+
+func (c config) getS3GroupUsersFilePath() string {
+	return fmt.Sprintf("%s%s", c.s3BaseDir, c.groupUsersFilename)
+}
+
+func readConfig() *config {
+	conf := &config{}
+
+	for _, e := range os.Environ() {
+		pair := strings.SplitN(e, "=", 2)
+		switch pair[0] {
+		case "groups_filename":
+			conf.groupsFilename = pair[1]
+		case "groupusers_filename":
+			conf.groupUsersFilename = pair[1]
+		case "s3_bucket":
+			conf.s3Bucket = pair[1]
+		case "s3_base_dir":
+			conf.s3BaseDir = pair[1]
+		case "s3_region":
+			conf.s3Region = pair[1]
+		case "user_pool_id":
+			conf.awsCognitoUserPoolID = pair[1]
+		}
+	}
```

Comment:
Can you update this to use the [github.com/kelseyhightower/envconfig](github.com/kelseyhightower/envconfig) library please? It will be consistent with the rest of our apps/libraries then. Here is an example of how to use that package:

https://github.com/ONSdigital/dp-identity-api/blob/e4c98e2179976d64526ab780da1f82025166001c/config/config.go#L12